graphene_django_plus/mutations.py

```diff
@@ -133,6 +134,11 @@ def _get_fields(model, only_fields, exclude_fields, required_fields):
                 description=field.help_text,
             )
         elif isinstance(field, (ManyToOneRel, ManyToManyRel)):
+            reverse_rel_include = graphene_django_plus_settings.MUTATIONS_INCLUDE_REVERSE_RELATIONS
+            # Checking whether it was globally configured to not include reverse relations
+            if isinstance(field, ManyToOneRel) and not reverse_rel_include and not only_fields:
```

Comment:
@justinask7 Adding 'unless the field is in only_fields' to the comment certainly makes the reading easier.